src/wallet/redux/reducers/responding.ts

```diff
@@ -1,6 +1,100 @@
 import { WalletState, RespondingState } from '../../states';
+import * as states from '../../states/responding';
+import * as challengeStates from '../../states/challenging';
+import * as runningStates from '../../states/running';
```

Comment:
Are you aware that these states are all exported from `../../states`, so you could just do the following if you wanted?:
```ts
import * as states from '../../states'
```

Reply:
I think I did this for clarity in development so that `states` would only contain the Responding states, and it would be clear when I was returning a state that is not a `responding` state